src/DCSAM.cpp

```diff
@@ -137,32 +145,36 @@ void DCSAM::updateContinuous() {
 void DCSAM::updateContinuousInfo(const DiscreteValues &discreteVals,
```

Comment:
Can remove preceding comment ("NOTE: for iSAM(2) requires special procedure [...]")